UITests/ToolbarTests.swift

```diff
@@ -28,7 +28,7 @@ class ToolbarTests: KIFTestCase, UITextFieldDelegate {
         let url = "\(localhostURL)/numberedPage.html?page=1"
 
         // URL without "http://".
-        let displayURL = "\(localhostURL)/numberedPage.html?page=1".substring(from: url.characters.index(url.startIndex, offsetBy: "http://".characters.count))
+        let displayURL = "\(localhostURL)/numberedPage.html?page=1".substring(from: url.characters.index(url.startIndex, offsetBy: "http://".count))
```

Comment:
I almost wonder if we shouldn't just hard-code `7` here?

Reply:
I like the "string".len syntax for clarity/safety